scripts/updateSchema/schema.graphql

```diff
@@ -0,0 +1,42 @@
+enum AccessLevel {
+  ADMIN
+  MANAGER
+  WORKER
+}
+
+type User {
+  name:String
+  email:String! @id
+  picture:String
```

Comment:
This is my POV (point of view) so take only in consideration if you thinks it will be a good idea.  Every single user will have different roles depending of the workspace. My idea is to have the roles on that.

type User {
    name: String
    email: String! @id
    picture: String
   roles: [UserRole] @hasInverse (field: user)
}

Reply:
One question, a role array is needed to know in which workspace that user is part of? I mean, only with a role array in Workspace we are allowing a user to have different roles depending on the Workspace or am I wrong?